docs/components/modeler/web-modeler/deploy-process-application.md

```diff
@@ -1,20 +1,33 @@
 ---
 id: deploy-process-application
-title: Deploy and run your process application
-description: You can deploy your process application to a cluster, and run your process application to test, debug, and observe how it performs in a live environment.
+title: Validate and deploy your process application
+description: Validate your process application in development before deploying it to testing, staging, or production.
 ---
 
 import DeployImg from './img/process-applications/define-stages-deploy.png'
 import RunProcessApplicationImg from './img/process-applications/run-process-application.png'
 import DeployErrorImg from './img/process-applications/deploy-error.png'
 
-You can deploy your process application to a cluster, and run your process application to test, debug, and observe how it performs in a live environment.
+You should validate your process application in development before deploying it to testing, staging, or production.
+
+## Validate your process application
+
+Use [play mode](/components/modeler/web-modeler/play-your-process.md) to validate your process application in development.
+
+1. Open the [main process](create-a-process-application.md#main-process).
+1. Select the **Play** tab to play the process application using your selected development cluster.
+1. Perform validation as required, for example, debug your process logic and test the process application.
+
+:::info
+To learn more about using Play for validation, see [play mode for rapid validation](/components/modeler/web-modeler/play-your-process.md)
+:::
 
 ## Deploy your process application
 
-To deploy your process application:
+Deploy your process application to testing, staging, or production, for example to run automated tests or make it available for testing.
```

Comment:
Linking back to the [deployment pipeline stages](https://docs.camunda.io/docs/next/components/modeler/web-modeler/process-application-pipeline/#deployment-pipeline-stages) might be useful here, and additional context:

"Once your application is validated, deploy it to the the cluster used in your [development lifecycle](https://docs.camunda.io/docs/next/components/modeler/web-modeler/process-application-pipeline/#deployment-pipeline-stages). The following steps are used to deploy your process application to any cluster environment, including production:"

Reply:
Thanks, feedback incorporated 👍